root/release/edit/information.tt

```diff
@@ -314,110 +314,253 @@
+        <p class="field-error">
+          <label>
+            <strong>
+              <input type="checkbox" data-bind="checked: confirmed" />
+              [% l("I confirm that this is the barcode as it appears on the release.") -%]
+            </strong>
+          </label>
+        </p>
+      <!-- /ko -->
     <!-- /ko -->
   </div>
 
+  <div class="bubble" data-bind="bubble: $root.packagingBubble">
+    <p>
+      [% l('The {doc|packaging} attribute describes the outermost packaging of the release.',
+           { doc => { href => doc_link('Release/Packaging'), target => '_blank' } }) %]
+    </p>
+    <p>
+      [% l('For digital media releases, this should usually be “None”.') %]
+    </p>
+  </div>
+
   <div class="bubble" data-bind="bubble: $root.annotationBubble">
     <p>
-      [% l('The annotation field functions like a miniature wiki.') %]
+      [% l('The {doc|annotation} field functions like a miniature wiki.',
+      { doc => { href => doc_link('Annotation'), target => '_blank' } }) %]
     </p>
     <p>
-      [% l('The purpose of this field is to add information that usually doesn\'t fit into the strict structural data schema of MusicBrainz (be it due to technical limitations that may be addressed later, or because the information in itself has to be free-text).') %]
+      [% l('The purpose of this field is to add information that usually
+            doesn’t fit into the strict structural data schema of MusicBrainz
+            (be it due to technical limitations that may be addressed later,
+            or because the information in itself has to be free-form text).') %]
     </p>
     <p>
-      [% l('This annotation will be published under an open license (<a href="{url}" target="_blank">CC BY-NC-SA 3.0</a>) and as such, it should not contain text that you don\'t have the right to release under that license. While you can quote a source to support a point you\'re making, you should never enter promotional texts or other artist or label-owned texts into the annotation.',
-      {url => "https://creativecommons.org/licenses/by-nc-sa/3.0/"}) %]
+      [% l('This annotation will be published under an open license
+            ({url|CC BY-NC-SA 3.0}) and as such, it should not contain text
+            that you don’t have the right to release under that license.
+            While you can quote a source to support a point you’re making,
+            you should never enter promotional text or other artist- or
+            label-owned text into the annotation.',
+           { url => { href => "https://creativecommons.org/licenses/by-nc-sa/3.0/", target => '_blank' } }) %]
     </p>
   </div>
 
   <div class="bubble" data-bind="bubble: $root.commentBubble">
     <p>
-      [% l('The disambiguation field is used to help users distinguish between identically named releases.') %]
+      [% l('The {doc|disambiguation} field is used to help users distinguish between identically-named releases.',
```

Comment:
I know this is mostly just keeping the existing text. Same as with the annotation, I do wonder whether we need to clarify that this can often be left blank for releases? (I've seen beginners entering whatever crap here)

Reply:
Yeah, same. FWIW, I'll probably give the artist form a similar treatment next, since I've noticed a lot of beginners treating the disambiguation field there like a `<meta name="description">` or `<meta name="keywords">` tag. :-/

Reply:
To be fair in this case the existing text may mislead new users (since I don't believe it's true that we want disambigs on all releases with the same name?)
Maybe:

The {doc|disambiguation} field is used to help users distinguish between otherwise identical releases.
It is fine to leave this field empty if other fields, such as title, label, date, etc, already distinguish this release.